.gitignore

```diff
@@ -1,27 +1,2 @@
-# Node rules:
-## Grunt intermediate storage (http://gruntjs.com/creating-plugins#storing-task-files)
-.grunt
-
-## Dependency directory
-## Commenting this out is preferred by some people, see
-## https://docs.npmjs.com/misc/faq#should-i-check-my-node_modules-folder-into-git
-node_modules
-
-# Book build output
-_book
-
-# eBook build output
-*.epub
-*.mobi
-*.pdf
-
-# Files generated by JetBrains IDEs, e.g. IntelliJ IDEA
-.idea/
-*.iml
-
-_preview
-_package
-.*swp
-
-# Docs is autogeneratde
-docs/
+# Site is autogenerated
+site/
```

Comment:
Are we keeping the generated content in the same branch or having a `source` branch from where the generated site is pushed into the publicly served branch (currently `master`)?

I believe we aim for the latter, in which case the `site` directory would not exist, right?

Reply:
The `site` directory must preexist.  `mkdocs build` does not create it.  For this piece I'm following what we do with the website where `_site` exists in feature branches but doesn't contain data or if it does it is ignored via `.gitignore`